### PR TITLE
Removing using namespace std;

### PR DESCRIPTION
**Pull request by QuantamHD**
Remove all usages of using namespace std; in the entire codebase. :)

### CODE REVIEW

**Comment by QuantamHD** [comment]
@maliberty @vvbandeira The clang format seems to be off. I can't replicate it on my side.

**Comment by QuantamHD** [comment]
nvm I just didn't push correctly

File: src/dpl/include/dpl/Graphics.h [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+#include "gui/gui.h"
+#include "odb/db.h"
+
+namespace dpl {
+
+class Opendp;
+struct Cell;
+
+// Decorates the gui::Renderer with DPL specific routines.
+class Graphics : public gui::Renderer
```
Comment (maliberty) [inline src/dpl/include/dpl/Graphics.h:39]: Why has this been moved to a public header?

File: src/drt/src/dr/FlexDR.cpp [hunk 2]
```diff
@@ -502,21 +506,22 @@ void FlexDR::searchRepair(const SearchRepairArgs& args)
...
 
-  vector<unique_ptr<FlexDRWorker>> uworkers;
+  std::vector<std::unique_ptr<FlexDRWorker>> uworkers;
   int batchStepX, batchStepY;
 
   getBatchInfo(batchStepX, batchStepY);
 
-  vector<vector<vector<unique_ptr<FlexDRWorker>>>> workers(batchStepX
-                                                           * batchStepY);
+  std::vector<std::vector<std::vector<std::unique_ptr<FlexDRWorker>>>> workers(
```
Comment (maliberty) [inline src/drt/src/dr/FlexDR.cpp:514]: I think a using std::vector in this cpp would help readability   [which isn't great to start with]

**Comment by maliberty** [comment]
With a broad change like this it is best to coordinate with other developers.  I imagine this may give @osamahammad21  many merge conflicts.

**Comment by QuantamHD** [comment]
You can merge his first. I don't mind rebasing.

**Comment by maliberty** [comment]
@osamahammad21 what PRs do you have remaining that should be merged first?

File: src/dpl/include/dpl/Graphics.h (hunk 1 above)
Comment (QuantamHD) [reply src/dpl/include/dpl/Graphics.h]: Fixed. Left over from different branch.

File: src/drt/src/dr/FlexDR.cpp (hunk 2 above)
Comment (QuantamHD) [reply src/drt/src/dr/FlexDR.cpp:514]: I'm going to say I agree, but that feels ill suited to this PR.

This particular case is a good example of the potential for making a large PR larger. The first std::vector in this type is being treated as a virtual 2D array. It's not clear to me that just adding a using declaration is enough to fix the readability issues. I think it's best to keep this PR focused on removing `using namespace std;`, and try evolving readability separately

**Comment by QuantamHD** [comment]
@maliberty @osamahammad21 Can you give me an O(5 day) merge window. One of the `using namespace std` does cause a compilation error on GCC 12.

**Comment by osamahammad21** [comment]
Merging this PR now will cause a lot of trouble as I have about 6 open PRs in the router. I would prefer if we delay it a week or two.

**Comment by QuantamHD** [comment]
@osamahammad21 Is it looking like we can merge this soon?

**Comment by maliberty** [comment]
Do you want to fix all the conflicts or abandon this?

**Comment by QuantamHD** [comment]
Happy to fix the conflicts if you guarantee a merge.

**Comment by maliberty** [comment]
> Happy to fix the conflicts if you guarantee a merge.

Ok, its as a good a time as any.

**Review by maliberty — APPROVED** [review]

**Pull request merged**